README.md

```diff
@@ -1,2 +1,63 @@
-# fxa-geodb
-Firefox Accounts IP geolocation library
+## Firefox Accounts GeoDB Repo for Geolocation based services
+Provides a wrapper around [node-maxmind] (https://github.com/runk/node-maxmind) for getting the location data in an easy-to-read format.
+
+### Getting started
+Clone/fork the repo and run `npm i`. Then, include the module in your source file, like so:
+
+```JavaScript
+var GeoDB = require('./fxa-geodb');
```

Comment:
According to @shane-tomlinson , Its going to be a node module. IIRC this was so that we can reuse this for other projects, such as customs-server.